src/notation/view/widgets/scoreproperties.cpp

```diff
@@ -245,31 +245,43 @@ bool ScorePropertiesDialog::save()
     const int idx = scrollAreaLayout->rowCount();
     QVariantMap map;
     for (int i = 0; i < idx; ++i) {
-        QLayoutItem* tagItem   = scrollAreaLayout->itemAtPosition(i, 0);
+        QLayoutItem* tagItem  = scrollAreaLayout->itemAtPosition(i, 0);
         QLayoutItem* valueItem = scrollAreaLayout->itemAtPosition(i, 1);
         if (tagItem && valueItem) {
-            QLineEdit* tag   = static_cast<QLineEdit*>(tagItem->widget());
+
+            // Label could be a line edit (for none standard properties) or a label widget (for standard properties)
+            QString tagText = "";
+            
+            QLineEdit* lineEditItem = dynamic_cast<QLineEdit*>(tagItem);
+            if (lineEditItem) {
+                tagText = lineEditItem->text();
+            }
+
+            QLabel* labelItem = dynamic_cast<QLabel*>(tagItem);
+            if (labelItem) {
+                tagText = labelItem->text();
+            }
+
             QLineEdit* value = static_cast<QLineEdit*>(valueItem->widget());
 
-            QString tagText = tag->text();
             if (tagText.isEmpty()) {
                 interactive()->warning(trc("notation", "MuseScore"),
                                        trc("notation", "Tags can't have empty names."));
-                tag->setFocus();
+                lineEditItem->setFocus();
```

Comment:
The crash can still occur here because of the null pointer. Also, I can't save new values of tags for some reason (always get this warning about empty names)...

Reply:
Rearranged the logic to make it more clearer. The validations would only ever fail on none standard properties - so this makes it clearer and the null checks now make more sense.